week2/index.js

```diff
@@ -65,17 +65,18 @@ const listTodoItem = (todoList) => {
     }
 };
 const showHelpMenu = (filename) => {
-    readTodosFile(filename).then((data) => {
-        console.log(data);
-    });
+    readTodosFile(filename)
+        .then((data) => {
+            console.log(data)
+                .catch((err) => {
```

Comment:
You have a bracket alignment problem here. You missed a bracket, so the code will behave as if you're trying to get the `.catch()` of the return value of `console.log()`, which obviously doesn't make any sense.

You've written:
```
readTodosFile(filename)
        .then((data) => {
            console.log(data)
                .catch((err) => {
```

But you meant to write:
```
readTodosFile(filename)
        .then((data) => {
            console.log(data)
        }).catch((err) => {
```

ie. the `.catch()` chains after the `.then()`.